package.json

```diff
@@ -150,6 +130,16 @@
                     "description": "Change mode between SSB or devsecops",
                     "default": "SSB",
                     "scope": "application"
+                },
+                "iiq-dev-accelerator.defaultExportObjectsPattern": {
+                    "type": "string",
+                    "description": "Define the pattern for the folder to export objects",
+                    "scope": "application"
+                },
+                "iiq-dev-accelerator.defaultExportObjectPattern": {
+                    "type": "string",
+                    "description": "Define the pattern for the folder to get an object",
+                    "scope": "application"
```

Comment:
Will default respect the old behavior?

Reply:
Yes

Reply:
The default behavior is ensured by the function `getDefaultExportedObjectFilename` and the test 
```
const pattern = this.getConfigKey(EXPORT_OBJECT_PATTERN);
objectName = this.sanitize(objectName);
if (typeof pattern !== "string" || pattern === "") {
  return this.getDefaultExportedObjectFilename(objectType, objectName);
]
```